test/python/Utils_t/Utilities_t.py

```diff
@@ -75,7 +74,7 @@ def safeStr(self):
         self.assertEqual(safeStr(""), "")
 
         for v in [[1, 2], {'x': 123}, set([1])]:
-            self.assertRaises(safeStr(v), '123')
+            self.assertRaises(ValueError, safeStr, v)
```

Comment:
This is fine. In general I think the 
'''
with self.assertRaises...
    safeStr(v)
```
form is more readable.